### DISCUSSION TITLE
Choose which conda environment Quarto uses to render files in VS Code

### DISCUSSION THREAD

**Discussion by ReedMerrill** [quarto-dev/quarto-cli#2469] [Q&A] [answered]
When I run `Quarto: Render PDF` in VS Code the first thing executed in the newly opened VS Code terminal is `conda activate test-env`. In my case `test-env` is the only conda environment (besides the base environment) that I have. Maybe I should also note that TinyTex is installed in my base environment, but not in `test-env`, so it isn't that Quarto is looking for a usable installation of TinyTex.

How can I choose which environment Quarto runs `render` in? I'm on Linux, and the .qmd file that I'm rendering has no relationship to the conda environment that Quarto uses to render it.

**Comment by msalazarcgeo** [comment]
I don't know if you solve the issue,  and I just answered if someone is looking on how to do it. This is the way I solved  you select the python environment , this can be done by  ' ctrl + P >Python: Select environment'. I had this issue once when my VS code was updated.

**Reply by cderv** [reply, marked as answer]
Thank you for adding this ! 

Yes, the environment Quarto Extension uses is related to the interpreter selected by Python extension in VSCODE. 

We probably also need to insure quarto CLI uses the Jupyter engine form this activated env.  (probably checking `CONDA_DEFAULT_ENV`)
